### PR TITLE
[docs] new sections and more

### PR DESCRIPTION
**Pull request by joeyparsons**
This includes the following:
- New introduction and feature areas under Getting Started
- Overviews in each major section

### CODE REVIEW

File: docs/first-service.md [hunk 1]
```diff
@@ -1,114 +1,127 @@
...
+effx resources are usually defined in `effx.yaml` files. You will learn how to define them and introduce automated workflows
+to keep your data up-to-date in these docs.
+
+The first step is to generate your first `effx.yaml` file for your initial service and then upload it via one of the methods below:
+
+## Adding your first service
+
+We've created a nifty [in-app YAML wizard](https://app.effx.com/yaml/wizard) to guide you through the process of creating your first service, but if you prefer to write the YAML yourself, below is the full definition of how to construct your own `effx.yaml`.  You can see a [full example](#effx-yaml-example) below.
+
+### effx-yaml definition
```
Comment (euriostigue) [inline docs/first-service.md:20]: This entire section is redundant with the "Resource Types" section.

I personally found this harder to read than the dynamic attribute component.

File: docs/first-team.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: Add your first Team
+---
+
+Much like services, adding teams is done by defining them via the GitOps workflow.  The simplest way to do this would be to create a `teamname.effx.yaml` file in a Git repo where it makes the most sense.
+
+Some of our users with monorepos will create a `teams` directory at the top-level of their repo and add a `teamname.effx.yaml` for each team within.  You can also create a `teams.effx.yaml` and define all of your teams in a single file as we support multiple objects per `effx.yaml` file.
+
+## Adding your first team
```
Comment (euriostigue) [inline docs/first-team.mdx:9]: This is another redundant section. I think it's better to re-use the Resource Types UI than restate it here.

File: docs/introduction.md [hunk 3]
```diff
@@ -3,30 +3,40 @@ title: Introduction
...
-The API provides flexibility for users that already have service metadata defined in other formats.
+Our API provides flexibility for users that already have service metadata defined in other formats.  Learn more about our API in the API Overview.
+
+### Use the CLI
+
+The `effx-cli` is the fundamental component behind most of our Automations. There are commands to help you work with `effx.yaml` files.
+
+It's encouraged to use the CLI on your local development environment for testing out your automated workflows.
+
+Learn more about effx-cli.
```
Comment (euriostigue) [inline docs/introduction.md:31]: Does this need a link?

File: docs/introduction.md [hunk 4]
```diff
@@ -3,30 +3,40 @@ title: Introduction
...
 
 ### Jump to Automations
 
-We offer convenient templates for enabling GitOps with popular CI systems.
+We offer convenient templates for enabling GitOps with popular version control and continuous integration systems.
 
-## Fetch your API Key
+Learn more about our Automations.
```
Comment (euriostigue) [inline docs/introduction.md:37]: Does this need a link?

File: docs/first-migration.mdx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+### Migration query defined
+
+The migration query is key to measuring the status of the migration.  There are two ways of defining the query:
+`Query by value`: Use this type of query when you want all of your tags to eventually be equal to a particular value.  For example, if you're moving all of your `deployment` (key) to `kubernetes` (value), you could define your Query as such:
+
+![Query by value](/img/migration-kubernetes.png)
+
+`Query by version`: Use this type of query when you want to compare semantic version tags.  For example, if you're wanting to upgrade all of your `go` services to `1.15.0` or greater, you can use this type of query:
+
+![Query by value](/img/migration-go.png)
```
Comment (euriostigue) [inline docs/first-migration.mdx:28]: The spacing here was condensed when I viewed this locally. I think adding some `<br>` will help.

File: docs/introduction.md [hunk 6]
```diff
@@ -3,30 +3,40 @@ title: Introduction
...
+## Accessing effx programmatically
 
-The CLI is the fundamental component behind most of our Automations. There are commands to help you work with `effx.yaml` files.
-It's encouraged to use the CLI on your local development environment for testing out your automated workflows.
+We provide numerous ways for you to incorporate your data with effx programmatically, whether it be through our API directly, using our `effx-cli` tool, or through Automations:
 
 ### Call our API
 
-The API provides flexibility for users that already have service metadata defined in other formats.
+Our API provides flexibility for users that already have service metadata defined in other formats.  Learn more about our API in the API Overview.
```
Comment (euriostigue) [inline docs/introduction.md:23]: Is there supposed to be a link here?

**Review by euriostigue — CHANGES_REQUESTED** [review]

File: docs/api-overview.mdx [hunk 7]
```diff
@@ -0,0 +1,24 @@
+---
+title: API Overview
```
[the remark this answers is not in the document]
Comment (joeyparsons) [reply docs/api-overview.mdx:2]: agreed!

**Review by euriostigue — APPROVED** [review]
Looks good to me!

**Pull request merged**